Sources/SplashImageGen/Extensions/CommandLine+Options.swift

```diff
@@ -46,6 +48,16 @@ extension CommandLine {
 
         return Font(path: path, size: size)
     }
+    
+    private static func resolveTheme(from defaults: UserDefaults) -> ThemeName {
```

Comment:
I've created a new type because I'm thinking about adding some other stuff that could potentially use it. One of them is providing user-generated themes as JSON files, so you don't need to recompile SplashImageGen when you want to apply different colors. In that case, it is handy to know if given string is a theme name or filename. I also don't like to compare raw string, prefer to wrap them into enums, but this is my personal approach. 

I think I can move `ThemeName` enum to `Theme` structure (so you could use as `Theme.Name`) and provide some static method on `Theme` like: `func get(name: String, font: Font) -> Theme` instead of using `func theme(withFont font: Splash.Font) -> Splash.Theme` from `ThemeName` enum. 
@JohnSundell  what do you think about this approach?